.travis.yml

```diff
@@ -3,7 +3,7 @@ language: python
 
 python:
   - "3.6"
-  - "3.7-dev"
+  - "3.7"
```

Comment:
Please don't change this in this PR.

Reply:
but 3.7-dev gave error of not existing putting the PR in red :(